### ISSUE TITLE
There should be a section in manifest for "checking_tool"

### ISSUE THREAD

**Issue by bspidel** [unfoldingWord/translationCore#5276]
## Summary

The checking tool should add or update the manifest to include its name, version and build 

## Motivation

Answers the question: What messed up my project?

## Specifications (If applicable)

checking_tools: [
   {
      name: tc,
      version: 1.0.2,
      build: 6ab77c0
   }
]

**Comment by benjore** [comment]
@jag3773 Can you speak to this? Is this within the spec? Should we wait until Scripture burrito comes out?

**Comment by jag3773** [comment]
Yes, that would be good to wait until Scripture Burrito comes out.  If you want to do something beforehand then you can write whatever you'd like into .apps/tc/ like we have documented here https://resource-container.readthedocs.io/en/latest/app_meta.html#translationcore .

**Comment by benjore** [comment]
Waiting for Scripture Burrito.

**Comment by elsylambert** [comment]
This feature is yet to be implemented.

**Comment by jag3773** [comment]
It is still true that as we move to Catalog Next and SB integration the metadata will include this information.  Note the [`generator` and `uploader` sections](https://github.com/bible-technology/scripture-burrito/blob/develop/docs/examples/artifacts/textTranslation.json#L6-L16).

cc @richmahn
